demisto_sdk/commands/format/update_playbook.py

```diff
@@ -39,14 +41,17 @@ def add_description(self):
         if 'description' not in set(self.data.keys()):
             click.secho('No description is specified for this playbook, would you like to add a description? [Y/n]',
                         fg='bright_red')
-            user_answer = ''
+            user_answer = 'y' if self.assume_yes else ''
```

Comment:
was this a bug?

Reply:
Yes it was another bug https://github.com/demisto/contribution-management/runs/4724862740?check_suite_focus=true#step:12:144